pipeline/src/constructors/make_inference_configs.jl

```diff
@@ -22,3 +22,29 @@ function make_inference_configs(pipeline::AbstractEpiAwarePipeline)
 
     return inference_configs
 end
+
+"""
+Create inference configurations for the given pipeline. This is the example method,
+which only returns a randomly selected inference configuration.
+
+# Arguments
+- `pipeline`: An instance of `EpiAwareExamplePipeline`.
+
+# Returns
+- An object representing the inference configurations.
+
+"""
+function make_inference_configs(pipeline::EpiAwareExamplePipeline)
+    gi_param_dict = make_gi_params(pipeline)
```

Comment:
do we not wnant to just call make_inference_configs(abstract method) here internally and then use rrand on it?